src/main.ts

```diff
@@ -28,18 +28,19 @@ Vue.use(Vuelidate)
 Vue.use(Affix)
 Vue.use(Vue2Filters) // needed by SbcFeeSummary
 
+const store = getVuexStore()
+
 // main code
 async function start () {
   // get config from environment
   const processEnvBaseUrl: string = process.env.BASE_URL // eg, /business/
   const windowLocationPathname = window.location.pathname // eg, /business/CP1234567/...
   const windowLocationOrigin = window.location.origin // eg, http://localhost:8080
   const applicationUrl = windowLocationOrigin + processEnvBaseUrl
-  setBaseRouteAndBusinessId(windowLocationPathname, processEnvBaseUrl, windowLocationOrigin) // may throw an error
 
-  // fetch the store first as it has no dependencies
-  const store = getVuexStore()
+  // first load the configuration, then set base route and check business id
   await store.dispatch('loadConfiguration', applicationUrl)
+  setBaseRouteAndBusinessId(windowLocationPathname, processEnvBaseUrl, windowLocationOrigin) // may throw an error
```

Comment:
setBaseRouteAndBusinessId() may throw an error, which we cannot recover from if we haven't loaded the configuration first.